test/spec/upload.js

```diff
@@ -54,8 +66,9 @@ describe("tus", function () {
 
     it("should upload a file", function (done) {
       var file = new Blob("hello world".split(""));
+      var host = "http://files.tus.io";
```

Comment:
Would you mind explaining the reasoning behind this change? I am usually a fan of DRYing up code but in tests I prefer to repeat constants. This makes adapting tests a bit harder but ensures that every change happens on purpose.

Reply:
the primary reason for this change was not necessarily to keep the code DRY, but it was so to make the host of each test case unique. The reason is so that the [nextTick function](https://github.com/tus/tus-js-client/pull/73/files/e2ceee428e0a35fa8bad046a71883888c88ab6b3#diff-893840c5db7894faca0e2e7ea5a7020dR1062) can correctly identify which new request belongs to a given test case.

And since the values of the `host` was changing by each test case, it became hard to repeat the correct host value within the test case

Reply:
Ok, so we have to remember to use a unique URL for every test case now if we use waitTillNextReq? If so, could you add this as a comment to the tests?